styles/index.css

```diff
@@ -0,0 +1,55 @@
+#layout {
+    width: 100vw;
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+    background-color: aliceblue;
+  }
+
+#heading {
+    grid-column: 1 / 3;
+    grid-row: 1 / 2;
+    text-align: center;
+}
+
+#city-title {
+    grid-column: 3/ 5;
+    grid-row: 1 / 2;
+    text-align: center;
+    padding-top: 9px;
+}
+
+#city {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+    text-align: center;
+}
+
+#up, #down {
+    height: 20px;
+}
+#temp-box {
+    grid-column: 1 / 3;
+    grid-row: 2 / 3;
```

Comment:
The temp-box appears to have been placed in the same part of the grid as the city input, and since the city _markup_ comes later in the html, it is effectively drawn on top of the buttons, making them unclickable with the mouse (though they can be tabbed to with the keyboard). This can be addressed by shifting the stuff in the left column down by a cell.